pkg/plugin/plugin.go

```diff
@@ -10,6 +9,11 @@ import (
 	"github.com/vincenzopalazzo/cln4go/plugin"
 )
 
+type PluginState struct {
+	Server   *gin.Engine
+	Password string
+}
+
 type LOGIN struct {
```

Comment:
```suggestion
type Login struct {
```

why you are writing all in upper ase case?